storage/Volume.cc

```diff
@@ -3188,6 +3205,14 @@ int Volume::doFstabUpdate( bool force_rewrite )
 		if (ret == 0)
 		    ret = extraFstabAdd(fstab, che);
 		fstab_added = true;
+
+		if (getMount() == "/" && getStorage()->isRootMounted())
+		{
+		    const CommitCallbacks* commit_callbacks = getStorage()->getCommitCallbacks();
+		    if (commit_callbacks)
```

Comment:
Sure, that is another possible implementation. But now a user can simply set the callback to nullptr to disable them.